dev/spec-setup.js

```diff
@@ -1,10 +1,15 @@
 /* eslint-disable global-require, no-console */
-
 var rowdy = require("rowdy");
 var MochaAdapter = rowdy.adapters.mocha;
 
+var SERVER_HOST = "127.0.0.1";
+var SERVER_PORT = "3000";
+global.TEST_FUNC_BASE_DIR = process.env.TEST_FUNC_BASE_DIR || "";
+global.TEST_FUNC_BASE_URL = process.env.TEST_FUNC_BASE_URL || "http://" + SERVER_HOST + ":" + SERVER_PORT + global.TEST_FUNC_BASE_DIR;
+
 var config = require("rowdy/config");
 config.options.driverLib = "webdriverio";
+config.settings.local.default.remote = { baseUrl: global.TEST_FUNC_BASE_URL };
```

Comment:
`webdriverio.remote(options)` is how you instantiate the webdriver and where all the options go, including baseUrl.

![image](https://cloud.githubusercontent.com/assets/848347/17188487/13bae582-53f2-11e6-9ccc-ddb04a720d86.png)

https://github.com/FormidableLabs/rowdy/blob/master/lib/client/base.js#L63
![image](https://cloud.githubusercontent.com/assets/848347/17188442/efff0dc6-53f1-11e6-86ff-6d5441722b0c.png)

https://github.com/FormidableLabs/rowdy/blob/master/lib/client/impl/webdriverio.js#L18
![image](https://cloud.githubusercontent.com/assets/848347/17188462/0152aef2-53f2-11e6-89a3-17785bed72c4.png)